index.html

```diff
@@ -166,5 +166,30 @@ <h3 class="skills"><img src="assets/Rectangle-3.png" alt="">Skills</h3>
             </section>
         </div>
     </div>
+    <section class="contact">
+        <div>
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form class="contact-form" method="post">
```

Comment:
- [ ] Kindly an action for your form and ensure your action value should be similar to ` https://formspree.io/f/{form_id}`, so that data collected is sent to your formspree account.